django/db/models/sql/compiler.py

```diff
@@ -1110,6 +1110,8 @@ def prepare_value(self, field, value):
                 )
             if value.contains_aggregate:
                 raise FieldError("Aggregate functions are not allowed in this query")
+            if value.contains_over_clause:
+                raise FieldError('Window expressions are not allowed in this query.')
```

Comment:
I only see 1 test for "Window expressions are not allowed in this query." but there are two places in this file with that message. Is a test missing?

Reply:
Probably. However, there's already `F() expressions can only be used to update, not to insert.` which catches the typical use of this expression. How about this?

```
def test_fail_insert(self):
    """Window expressions can't be used in an INSERT statement."""
    msg = 'Window expressions are not allowed in this query'
    with self.assertRaisesMessage(FieldError, msg):
        Employee.objects.create(
            name='Jameson', department='Management', hire_date=datetime.date(2007, 7, 1),
            salary=Window(expression=Sum(Value(10000))),
        )
```